lib/main_common.pm

```diff
@@ -998,6 +998,10 @@ sub load_inst_tests {
         loadtest "installation/secure_boot";
     }
     if (installyaststep_is_applicable()) {
+        # Do not repeat step when change_desktop has been executed
+        if (check_var("ENABLE_DEPENDENCY_RESOLVER", "1")) {
```

Comment:
Do you already have a list of test suites which need this variable to be set? I guess we can set them in advance and then merge.

Reply:
Some of them:
textmode
allmodules+allpatterns
minimal+base
minimal+base@s390x-zVM-vswitch-l2
minimal+base@s390x-zVM-vswitch-l3
minimal+base@svirt-hyperv
minimal+base@svirt-hyperv-uefi
minimal+base@svirt-xen-hvm
minimal+base@svirt-xen-pv
offline_sles12sp3_pscc_all_full_zVM_preparation
offline_sles12sp4_pscc_all_full_zVM_preparation
offline_sles15_pscc_basesys+srv_all_full_zVM_preparation